buildpack.toml

```diff
@@ -11,5 +11,14 @@ keywords = ["procfile", "processes"]
 [[stacks]]
 id = "*"
 
+[[stacks]]
+id = "heroku-18"
+
+[[stacks]]
+id = "heroku-20"
+
+[[stacks]]
+id = "heroku-20"
```

Comment:
```suggestion
id = "heroku-22"
```

Reply:
Fixed in https://github.com/heroku/procfile-cnb/pull/55/commits/c31c97740df4b19bdd44919330e676b062f97113.